cosmos/profiles/snowflake/user_privatekey.py

```diff
@@ -74,11 +74,3 @@ def profile(self) -> dict[str, Any | None]:
 
         # remove any null values
         return self.filter_null(profile_vars)
-
-    def transform_account(self, account: str) -> str:
```

Comment:
Should we also remove this method from the other 3 modules `user_encrypted_privatekey_env_variable.py`, `user_pass.py` and `user_encrypted_privatekey_file.py`?

Reply:
Done in 8abaee64c08be1d6fa98ea5fd430170f6daa886b